### ISSUE TITLE
remove gplay beta channel

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-android#2312]
we do not add betas currently, however, it is offered for "joining" on the normal page.

it offers somehow the recent versions, while the version shown in gplay console is 1.2.1, it is weird, we should just remove that option.

we hesited to remove the channel, however, as it seems unclear what happens with existing users - which may be quite some as we only offered betas for more than a year (all versions below 1.0)

![Screenshot 2022-06-02 at 12 19 32](https://user-images.githubusercontent.com/9800740/171609215-f76e009c-ccb7-49dc-9a72-7604c2fadca1.png)

what happens to existing users?

**Comment by r10s** [comment]
nb: there are quite some ppl on the beta track:

<img width="791" alt="Screenshot 2022-06-19 at 12 17 50" src="https://user-images.githubusercontent.com/9800740/174476293-9384d5c7-de41-4f87-9ff6-ce4464fbd72d.png">

**Comment by adbenitez** [comment]
can't we remove the beta channel? I bet if we remove it people will switch automatically to normal channel, and then they can give us stars :relaxed:

**Comment by r10s** [comment]
@adbenitez paused the beta track as initially suggested, however, we did not find any button to disable it. 

so, there is still a "Join the beta" button on the page (as on tons of other gplay pages, however, not on all, eg. "Wasavi" (random hit) does not have it). but we did not find any option/hint on so or reddit, so we gave up for now.

_might_ be, it takes some time until the button is actually removed, _might_ be this is done after the next release ...

nb: if you join the beta program (before or after the disabling of the track), you get the most recent 1.40, so, it seems that at least ppl never get to the super-old 1.2.1
